### PR TITLE
updated caption on 06-harshtandon23 file

### PR DESCRIPTION
**Pull request by harshtandon23**

### CODE REVIEW

File: _slides/06-harshtandon23.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Never settle for just one scoop ;)
```
Comment (bterrific2008) [inline _slides/06-harshtandon23.md:6]: Maybe use the unicode character 😉

Comment (harshtandon23) [reply _slides/06-harshtandon23.md:6]: Cool

Comment (harshtandon23) [reply _slides/06-harshtandon23.md:6]: U+1F44D

Comment (harshtandon23) [reply _slides/06-harshtandon23.md:6]: 👍

**Pull request merged**